test/hotspot/gtest/nmt/test_vmatree.cpp

```diff
@@ -725,18 +986,2270 @@ TEST_VM_F(NMTVMATreeTest, TestConsistencyWithSimpleTracker) {
+  const VMATree::RequestInfo       ReserveRequest{0, a, Rs, ReqTag, s2, false};
+    //            1         2         3         4         5
+    //  012345678901234567890123456789012345678901234567890
+    //  rrrrrrrrrr.....CCCCCCCCCC.....rrrrrrrrrr
+  const VMATree::RequestInfo        CommitRequest{0, a,  C, ReqTag, s2, false};
+  const VMATree::RequestInfo      UncommitRequest{0, a, Rs, mtNone, ES, true};
+  const VMATree::RequestInfo CopyTagCommitRequest{0, a,  C, ReqTag, s2, true};
+                              //  existing state           request              expected state     expected diff
+                              // st   tag    stacks                           st   tag    stacks   reserve  commit
+                              // --  ------  ------  ----------------------   --  ------  ------   -------  -------
+  UpdateCallInfo  call_info[]={{{Rl, mtNone, ES, ES},        ReleaseRequest, {Rl, mtNone, ES, ES}, {0,  0}, {0,  0}},
+                               {{Rl, mtNone, ES, ES},        ReserveRequest, {Rs, ReqTag, s2, ES}, {0,  a}, {0,  0}},
+                               {{Rl, mtNone, ES, ES},         CommitRequest, { C, ReqTag, s2, s2}, {0,  a}, {0,  a}},
+                               {{Rl, mtNone, ES, ES},  CopyTagCommitRequest, { C, mtNone, s2, s2}, {0,  a}, {0,  a}},
+                               {{Rl, mtNone, ES, ES},       UncommitRequest, {Rl, mtNone, ES, ES}, {0,  0}, {0,  0}},
+                               {{Rs,   mtGC, s0, ES},        ReleaseRequest, {Rl, mtNone, ES, ES}, {-a, 0}, {0,  0}},
+                               {{Rs,   mtGC, s0, ES},        ReserveRequest, {Rs, ReqTag, s2, ES}, {-a, a}, {0,  0}}, // diff tag
+                               {{Rs, mtTest, s0, ES},        ReserveRequest, {Rs, ReqTag, s2, ES}, {0,  0}, {0,  0}}, // same tag
+                               {{Rs,   mtGC, s0, ES},         CommitRequest, { C, ReqTag, s0, s2}, {-a, a}, {0,  a}},
+                               {{Rs,   mtGC, s0, ES},  CopyTagCommitRequest, { C,   mtGC, s0, s2}, {0,  0}, {0,  a}},
+                               {{Rs,   mtGC, s0, ES},       UncommitRequest, {Rs,   mtGC, s0, ES}, {0,  0}, {0,  0}},
+                               {{ C,   mtGC, s0, s1},        ReleaseRequest, {Rl, mtNone, ES, ES}, {-a, 0}, {-a, 0}},
+                               {{ C,   mtGC, s0, s1},        ReserveRequest, {Rs, ReqTag, s2, ES}, {-a, a}, {-a, 0}}, // diff tag
+                               {{ C, mtTest, s0, s1},        ReserveRequest, {Rs, ReqTag, s2, ES}, {0,  0}, {-a, 0}}, // same tag
+                               {{ C,   mtGC, s0, s1},         CommitRequest, { C, ReqTag, s0, s2}, {-a, a}, {-a, a}},
+                               {{ C,   mtGC, s0, s1},  CopyTagCommitRequest, { C,   mtGC, s0, s2}, {0,  0}, {-a, a}},
+    //            1         2         3         4         5
+    //  012345678901234567890123456789012345678901234567890
+    //  rrrrrrrrrr..........rrrrrrrrrr
```

Comment:
What is this comment supposed to show?

I cleaned up, some comments and created a branch https://github.com/openjdk/jdk/compare/master...gerard-ziemski:jdk:gerard-8351661 in case you want to see it.